### PR TITLE
Add register and list command

### PR DESCRIPTION
**Pull request by potsbo**
## WHY

We need register and list command to read and write our account names.

## WHAT

- [x] Add DynamoDB model with `list` and `add` capability
- [x] Add User model with GitHub username
- [x] Implement `register` and `list` command using those above

### Test
```console
potsbo@quorra-INS-% envchain wtd ./bin/developers-account-mapper register potsbo potsbo
user potsbo:@potsbo added.
potsbo@quorra-INS-% envchain wtd ./bin/developers-account-mapper list
potsbo:@potsbo
```

We do need automatic test and write ones.
Planned to introduce in a later PR.
This is why I haven't done anything on `*_test.go`

### CODE REVIEW

File: command/list.go [hunk 1]
```diff
@@ -2,14 +2,26 @@ package command
 
 import (
 	"strings"
+	"github.com/wantedly/developers-account-mapper/store"
```
Comment (dtan4) [inline command/list.go]: Please separate third-party resource imports from standard libraries.

```go
 import (
  	"strings"	

  	"github.com/wantedly/developers-account-mapper/store"	
)
```

File: command/register.go [hunk 2]
```diff
@@ -2,14 +2,39 @@ package command
...
+	} else {
+		log.Println(c.Help())
+		return 1
+	}
+
+	s := store.NewDynamoDB()
+
+	user := models.NewUser(loginName, githubUsername)
+	err := s.AddUser(user)
+	if err != nil {
```
Comment (dtan4) [inline command/register.go]: These can be written:

```go
if err := s.AddUser(user); err != nil {
```

File: command/register.go [hunk 3]
```diff
@@ -2,14 +2,39 @@ package command
...
+		log.Println(err)
+		return 1
+	}
+	fmt.Printf("user %v added.\n", user)
```
Comment (dtan4) [inline command/register.go]: `%v` is not preferable for success-case output. Please consider using `%s` or `%q` (double-quoted).

File: glide.yaml [hunk 4]
```diff
@@ -1,3 +1,8 @@
 package: github.com/wantedly/developers-account-mapper
 import:
 - package: github.com/mitchellh/cli
+- package: github.com/aws/aws-sdk-go
+  subpackages:
+  - aws
+  - aws/session
+  - aws/service/dynamodb
```
Comment (dtan4) [inline glide.yaml:8]: To keep every build reproducible, please commit `glide.lock` in this repository.

**Review by dtan4 — CHANGES_REQUESTED** [review]

**Comment by potsbo** [comment]
@dtan4 Is [this](https://github.com/wantedly/developers-account-mapper/pull/6#pullrequestreview-13492714) an accepted style to write Go? Any reference?

File: glide.yaml (hunk 4 above)
Comment (potsbo) [reply glide.yaml:8]: @dtan4 Running `glide update` lists dozens of lines of subpackages in glied.yaml, while [glide.lock of slack-mention-converter](https://github.com/wantedly/slack-mention-converter/blob/master/glide.lock) is so clean
Is this what is should be? Am I doing anything wrong?

Comment (dtan4) [reply glide.yaml:8]: It's not wrong. Maybe slack-mention-converter one is that we have forgotten doing `glide update`. No matter how many lines added, it's ok. Please see https://github.com/dtan4/s3url/blob/master/glide.lock .

Actually, we just want to keep __revision__ of dependencies.

Comment (potsbo) [reply glide.yaml:8]: All right then I believe it's ready.

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**